src/components/SearchForm.vue

```diff
@@ -18,7 +18,7 @@ export default {
   props: {
     isDisabled: {
       type: Boolean,
-      required: true,
+      required: false,
```

Comment:
če ni required, predlagam da dodaš nek default value + required: false je že default vrednost in ne rabiš explicitno pisat